memcached/content.md

```diff
@@ -12,12 +12,31 @@ Memcached's APIs provide a very large hash table distributed across multiple mac
 $ docker run --name my-memcache -d %%IMAGE%%
 ```
 
-## Setting Memory Usage
+### Configuration
+
+To customize the configuration of the memcached server, first obtain the upstream configuration options from the container:
+
+```console
+$ docker run --rm %%IMAGE%% -h
+```
+
+Using `docker run`:
 
 ```console
-$ docker run --name my-memcache -d %%IMAGE%% memcached -m 64
+docker run --name my-memcache -d %%IMAGE%% memcached:alpine --memory-limit=64
 ```
 
-This would set the Memcached server to use 64 megabytes for storage.
+or using Docker Compose:
+
+```yaml
+services:
+  memcached:
+    image: %%IMAGE%%:alpine
+    command: [
+      "--conn-limit=1024",
+      "--memory-limit=64",
+      --threads=4"
+    ]
+```
```

Comment:
I think the trailing comma situation makes the `[]` syntax here a little harder to maintain correctly (and we're missing a `"` :sweat_smile:), so I think we should swap this to the more standard "list" syntax, even though `- --xxx` looks kind of funny:

````suggestion
or using Docker Compose:

```yaml
services:
  memcached:
    image: %%IMAGE%%
    command:
      - --conn-limit=1024
      - --memory-limit=64
      - --threads=4
```
````

Alternatively, we could use the single-line format:

````suggestion
or using Docker Compose:

```yaml
services:
  memcached:
    image: %%IMAGE%%
    command: --conn-limit=1024 --memory-limit=64 --threads=4
```
````

Reply:
IMO, the list is better if you use more options